esphome/components/mhz19/sensor.py

```diff
@@ -68,6 +72,9 @@ async def to_code(config):
     if CONF_AUTOMATIC_BASELINE_CALIBRATION in config:
         cg.add(var.set_abc_enabled(config[CONF_AUTOMATIC_BASELINE_CALIBRATION]))
 
+    if CONF_WARMUP_TIME in config:
+        cg.add(var.set_warmup_seconds(config[CONF_WARMUP_TIME]))
+
```

Comment:
There is a default, so it is always there
```suggestion
    cg.add(var.set_warmup_seconds(config[CONF_WARMUP_TIME]))

```